### ISSUE TITLE
Add public stream of photos uploaded or some browsable repository of uploads

### ISSUE THREAD

**Issue by rejon** [newpalmyra/newpalmyra.org#67]

**Comment by rejon** [comment]
We understand how openclipart, flickr, fontlibrary works, should just do the same as a feature of the project.

**Comment by christopheradams** [comment]
- [ ] create a #NewPalmyra Flickr account
- [ ] upload backlog of photos
- [ ] update photo and album descriptions, and make photos public
- [ ] create Flickr app and API key
- [ ] add Ruby script to pull Flickr albums down into Jekyll `_data` files (might want to have a tag to identify which photos should flow onto the main site)
- [ ] add layout/templates to render the photo data
- [ ] create a new task for handling new uploads

**Comment by rejon** [comment]
@christopheradams how is this going? Ready to flow. The installation is on target, and projection too. Need fuel.

**Comment by rejon** [comment]
@barrythrew I closed the flickr account item above, that is correct right?

**Comment by rejon** [comment]
Ok, where these photos at? I feel like deleting the entire website today.

**Comment by christopheradams** [comment]
https://www.flickr.com/photos/newpalmyra/albums

**Comment by rejon** [comment]
GREAT!

On Thu, Apr 20, 2017 at 4:33 PM, Christopher Adams <notifications@github.com

**Comment by rejon** [comment]
COOL, shared on social media.

**Comment by rejon** [comment]
Some of these are done? Christopher can you reword this to be more accurate?

**Comment by christopheradams** [comment]
Yep, I think we can close this one now.
